### PR TITLE
[GFTCodeFixer]:  Update on src/main/java/com/scalesec/vulnado/Cowsay.java

### PR DESCRIPTION
**Pull request by julioarruda**

### CODE REVIEW

File: src/main/java/com/scalesec/vulnado/Cowsay.java [hunk 1]
```diff
@@ -21,8 +24,10 @@ public static String run(String input) {
         output.append(line + "\n");
       }
     } catch (Exception e) {
-      e.printStackTrace();
+      LOGGER.severe("Error executing cowsay command: " + e.getMessage());
     }
     return output.toString();
```
Comment (julioarruda) [inline src/main/java/com/scalesec/vulnado/Cowsay.java:29]: **[F1]** 🔴 **Incomplete sanitizeInput implementation breaks compilation**

The newly added sanitizeInput method has no implementation or return statement, and the class braces are malformed. The file will not compile in its current state.

```
Implement the method and fix the class structure, for example:
```java
private static String sanitizeInput(String input) {
    return input.replaceAll("[^a-zA-Z0-9\\s]", "");
}
```
Ensure the method and class are properly closed with matching braces.
```

File: src/main/java/com/scalesec/vulnado/Cowsay.java [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 
 public class Cowsay {
+  private Cowsay() {}
+  private static final Logger LOGGER = Logger.getLogger(Cowsay.class.getName());
   public static String run(String input) {
     ProcessBuilder processBuilder = new ProcessBuilder();
-    String cmd = "/usr/games/cowsay '" + input + "'";
-    System.out.println(cmd);
+    String cmd = "/usr/games/cowsay " + sanitizeInput(input);
+    LOGGER.info(cmd);
```
Comment (julioarruda) [inline src/main/java/com/scalesec/vulnado/Cowsay.java:13]: **[F2]** 🟠 **Command execution still relies on shell invocation**

Although sanitization was introduced, the code still invokes `bash -c`, which keeps shell parsing in the execution path and increases command injection risk if sanitization is bypassed or incomplete.

```
Avoid shell execution entirely by passing arguments directly to ProcessBuilder:
```java
processBuilder.command("/usr/games/cowsay", sanitizeInput(input));
```
This removes the need for `bash -c`.
```

File: src/main/java/com/scalesec/vulnado/Cowsay.java [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
 import java.io.InputStreamReader;
 
 public class Cowsay {
+  private Cowsay() {}
+  private static final Logger LOGGER = Logger.getLogger(Cowsay.class.getName());
   public static String run(String input) {
     ProcessBuilder processBuilder = new ProcessBuilder();
-    String cmd = "/usr/games/cowsay '" + input + "'";
-    System.out.println(cmd);
+    String cmd = "/usr/games/cowsay " + sanitizeInput(input);
```
Comment (julioarruda) [inline src/main/java/com/scalesec/vulnado/Cowsay.java:12]: **[F3]** 🟠 **No tests found for command sanitization behavior**

Repository search found no tests covering `Cowsay.run` or the new sanitization logic. The PR changes command construction and logging behavior without validation that malicious or malformed input is handled safely.

```
Add unit tests covering normal input, shell metacharacters, empty input, and logging/error paths for `Cowsay.run` and `sanitizeInput`.
```
